polaris.shopify.com/src/components/GlobalSearch/GlobalSearch.tsx

```diff
@@ -77,16 +77,17 @@ function GlobalSearch({}: Props) {
   useEffect(() => {
     document.addEventListener("keydown", (event) => {
       const searchbar = document.getElementById(globalSearchID);
-      let isKKey = "/";
-      if (isKKey) {
+      let isSlashKey = event.key === "/";
+      if (isSlashKey) {
         event.preventDefault();
         openMenu();
         if (searchbar !== null) {
           searchbar.focus();
         }
       }
     });
-  });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
If you don't include a dependency array, the hook will be run every time the component rerenders. In this case, it means that the `addEventListener` will be bound over and over again.